manifests/m/mercedesclk320/SamsungRemote/1.0/mercedesclk320.SamsungRemote.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: mercedesclk320.SamsungRemote
+PackageVersion: 1.0
+Installers:
+- InstallerUrl: https://forum.xda-developers.com/attachments/samsungremote-zip.503985/

```

Comment:
Please provide documentation showing that the publisher has listed this forum as the official mirror

Reply:
This publisher's name prevents internet searching, as all results are consumed by SEO for the similarly-named product. Please provide more information about the publisher or developer.